### DISCUSSION TITLE
Dust luminosity

### DISCUSSION THREAD

**Discussion by Bastiebb** [mboquien/cigale#91] [Q&A] [answered]
In CIGALE I use the estimation of dust luminosity, but I am wondering what the range of the dust luminosity is. So what is the spectrum of the dust luminosity given by CIGALE?

**Comment by mboquien** [comment, marked as answer]
This depends on the underlying dust model. All the dust emission is integrated. One way to do this is to look at the dust components of best-fit files to check the wavelength range over which they are non-zero.
